tests/polynomial/lower_mul.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: heir-opt --polynomial-to-standard %s | FileCheck %s
+// RUN: heir-opt --polynomial-to-standard --split-input-file %s | FileCheck %s
```

Comment:
A little tip: while using split-input-file is fine, if you don't want to use this, then what you do is use `CHECK-LABEL` instead of `CHECK` when matching function names (and use `CHECK-SAME` if you need to capture regexes of function arguments on the same line), and that creates a logical separation between test cases that FileCheck understands. A decent example is here: https://github.com/google/heir/blob/1125bfd29a65f5c31dddbabf84749688a70615e9/tests/secret/distribute_generic.mlir#L3-L4

Reply:
Nice tip! In this case I used `--split-input-file` since I changed the coefficient modulus and to avoid having to create new values with different names. But maybe that would create confusion when reading the testcase